src/main/java/core/basesyntax/service/impl/ReportCreatorImpl.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.FruitsDao;
+import core.basesyntax.service.ReportCreator;
+import java.util.ArrayList;
+import java.util.Map;
+
+public class ReportCreatorImpl implements ReportCreator {
+    private String header;
+    private FruitsDao fruitsDao;
+
+    public ReportCreatorImpl(String header, FruitsDao fruitsDao) {

```

Comment:
don't need to pass a header as a parameter, just create a constant with this value here, in this class

Reply:
This was a recommendation from first review - initialize header with constructor.
![Screenshot_3](https://user-images.githubusercontent.com/97477837/184864760-6967ecc5-9812-4d90-9e58-0b8ed965edbe.png)
.